composer.json

```diff
@@ -19,7 +19,9 @@
     "require-dev": {
         "phpunit/phpunit": "^6.3",
         "phpunit/phpunit-mock-objects": "!=3.2.4,!=3.2.5",
-        "symfony/console": "2.*||^3.0"
+        "symfony/console": "2.*||^3.0",
+        "doctrine/coding-standard": "^1.0",
+        "squizlabs/php_codesniffer": "^3.0"
```

Comment:
consider adding `sort-packages: true` to the `config` section, it will make diffs like this one less confusing.